### PR TITLE
PAYEE -589 FE Library Upgrade - WhatDoYouWantToDoTileView 

### PR DESCRIPTION
**Pull request by masahmad**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/views/WhatDoYouWantToDoTileView.scala.html [hunk 1]
```diff
@@ -41,22 +41,22 @@
   }
 }
 
+
 @mainTemplate(
     title = messages("your.paye.income.tax.overview"),
     authedUser = Some(user),
-    optimizelyAudience = optimizelyAudience,
-    showUrBannerOverride = true
```
Comment (brendanstraughan) [inline app/views/WhatDoYouWantToDoTileView.scala.html:48]: This has removed the UR banner from the main page, is this intended?

File: app/views/includes/forms/card.scala.html [hunk 2]
```diff
@@ -19,26 +19,25 @@
...
 <div class="@cardClass" id="@id">
 
-  <div class="card-body @if(url){active}">
-
-    <@headingTag class="heading-small card-heading">
+  <div class="card-body cardSize @if(url){active}">
+    <@headingTag class="govuk-heading-s card-heading">
       @if(url){
-        <a class="card-link" href="@url">
+        <a class="card-linkxx" href="@url">
```
Comment (brendanstraughan) [inline app/views/includes/forms/card.scala.html:33]: is this meant to say linkxx?

File: app/views/WhatDoYouWantToDoTileView.scala.html [hunk 3]
```diff
@@ -41,22 +41,22 @@
   }
 }
 
+
 @mainTemplate(
     title = messages("your.paye.income.tax.overview"),
     authedUser = Some(user),
-    optimizelyAudience = optimizelyAudience,
```
Comment (brendanstraughan) [inline app/views/WhatDoYouWantToDoTileView.scala.html:47]: is it ok to remove this?

Comment (masahmad) [reply app/views/WhatDoYouWantToDoTileView.scala.html:47]: there is no reference to   "optimizelyAudience" in the old maintemplate even though we are passing it in as a param so it looks like its unused.

File: app/views/includes/forms/card.scala.html (hunk 2 above)
Comment (masahmad) [reply app/views/includes/forms/card.scala.html:33]: yes thnks  this is a typo - class has been removed

File: app/views/WhatDoYouWantToDoTileView.scala.html (hunk 1 above)
Comment (masahmad) [reply app/views/WhatDoYouWantToDoTileView.scala.html:48]: good spot - we need  the banner - however just noticed we dont have this feature in the new main template...

File: app/views/WhatDoYouWantToDoTileView.scala.html [hunk 4]
```diff
@@ -41,22 +41,22 @@
...
 ) {
 
     <header class="page-header">
-        <h1 class="heading-xlarge" id="whatDoYouWantToDoTitle">@messages("your.paye.income.tax.overview")</h1>
+        <h1 class="govuk-heading-xl" id="whatDoYouWantToDoTitle">@messages("your.paye.income.tax.overview")</h1>
     </header>
 
-    <div class="grid-row">
+    <div class="govuk-grid-row">
         <div class="grid">
```
Comment (sergi-hmrc) [inline app/views/WhatDoYouWantToDoTileView.scala.html:57]: is this grid necessary?

File: app/views/WhatDoYouWantToDoTileView.scala.html (hunk 1 above)
Comment (brendanstraughan) [reply app/views/WhatDoYouWantToDoTileView.scala.html:48]: Going to be resolved as part of another ticket

File: app/views/WhatDoYouWantToDoTileView.scala.html [hunk 5]
```diff
@@ -41,22 +41,22 @@
...
 
     <header class="page-header">
-        <h1 class="heading-xlarge" id="whatDoYouWantToDoTitle">@messages("your.paye.income.tax.overview")</h1>
+        <h1 class="govuk-heading-xl" id="whatDoYouWantToDoTitle">@messages("your.paye.income.tax.overview")</h1>
     </header>
 
-    <div class="grid-row">
-        <div class="grid">
+    <div class="govuk-grid-row">
+        <div>
```
Comment (brendanstraughan) [inline app/views/WhatDoYouWantToDoTileView.scala.html:57]: can we now remove this div since there is no class

**Review by brendanstraughan — APPROVED** [review]

File: app/views/WhatDoYouWantToDoTileView.scala.html (hunk 4 above)
Comment (masahmad) [reply app/views/WhatDoYouWantToDoTileView.scala.html]: removed

File: app/views/MainTempleteNew.scala.html [hunk 6]
```diff
@@ -77,19 +77,20 @@
...
         additionalScriptsBlock: Option[Html] = None,
         beforeContentBlock: Option[Html] = None,
-        headerContainerClasses: String = Header.defaultObject.containerClasses,
         pagePrintable: Boolean = false,
         pagePrintName: Option[String] = None,
         showPtaAccountNav: Boolean = true,
-        formForErrorSummary: Option[Form[_]] = None
+        formForErrorSummary: Option[Form[_]] = None,
+        nonce: Option[String] = None,
+        showUrBannerOverride: Boolean = false
```
Comment (sergi-hmrc) [inline app/views/MainTempleteNew.scala.html:88]: i think we should start grouping some parameters like backLink, pagePrint etc

**Review by brendanstraughan — COMMENTED** [review]
When checking locally the boxes dont work for me. There is no highlighting and the only clickable part is the text?

edit: Reddy is confirming if this is correct behaviour

**Review by brendanstraughan — APPROVED** [review]

**Pull request merged**